content/en/docs/concepts/storage/volumes.md

```diff
@@ -74,7 +74,8 @@ Kubernetes supports several types of Volumes:
    * [csi](#csi)
    * [downwardAPI](#downwardapi)
    * [emptyDir](#emptydir)
-   * [fc (fibre channel)](#fc)
+   * [fc (fibre channel)](#fc)flexVolume
```

Comment:
?

Reply:
Huh. No idea what happened here.

Reply:
Should be fine now. Thanks for reviewing.